src/test/java/teammates/test/cases/ui/browsertests/FeedbackRubricQuestionUiTest.java

```diff
@@ -187,15 +186,21 @@ private void testNewRubricQuestionFrame() {
         assertTrue(feedbackEditPage.verifyNewRubricQuestionFormIsDisplayed());
     }
     
-    private void testInputValidationForRubricQuestion() {
+    public void testInputValidation() {
         
         ______TS("empty question text");
 
         feedbackEditPage.clickAddQuestionButton();
         assertEquals(Const.StatusMessages.FEEDBACK_QUESTION_TEXTINVALID, feedbackEditPage.getStatus());
     }
+    
+    public void testCustomizeOptions() {
+        
+        // TODO somebody do this?
```

Comment:
Actually, lots of parts of testEditQuestionActions and testCustomizeOptions are very interchangeable.
I need to put the method there because it's part of the abstract class. Of course it can be left empty.